### PR TITLE
Add JSON parsing errors exception handle

### LINKED ISSUE #18: Add JSON parsing error handling
**Descriptions**
When a user uploads a bookmarks JSON file, [this](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/options/options.js#L45) event listener parses the file.

An error occurs when an invalid JSON file is uploaded but no message or feedback is given to the user.

We need to add JSON parsing error handling and show an appropriate message like `Not valid json file` as a notification in the event listener.

### PR DESCRIPTION
**Pull request by manisacharya**
Fixes #18 

**Description**
- Add JSON parsing errors exception handle

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

### CODE REVIEW

File: src/options/options.js [hunk 1]
```diff
@@ -86,16 +84,13 @@ document.getElementById('vocab-tabbed-header').addEventListener('click', (e) =>
...
-      document.getElementById('vocab-tabbed-contents').children,
-      (element) => {
-        element.classList.remove('active');
-        if (element.getAttribute('data-content-no') === tabNo) {
-          // saving the target content div
-          targetContentDiv = element;
-        }
-      },
-    );
+    Array.prototype.forEach.call(document.getElementById('vocab-tabbed-contents').children, element => {
```
Comment (makkoncept) [inline src/options/options.js]: This project uses a column width of 120 charachters. It seems you are using 80 characters per line. We don't want to wrap code before 120 charachters.

File: src/options/options.js [hunk 2]
```diff
@@ -51,31 +49,31 @@ elements.importBookmarksButton.addEventListener('click', () => {
...
+      } catch (error) {
+        showNotification('This is not a valid JSON', 'negative', 'snackbar-options');
       }
     };
   }
 });
 
 // tab switching logic
-document.getElementById('vocab-tabbed-header').addEventListener('click', (e) => {
+document.getElementById('vocab-tabbed-header').addEventListener('click', e => {
```
Comment (makkoncept) [inline src/options/options.js]: You have removed params around arrow functions. This project uses eslint(Airbnb style) for linting and this violates [arrow-params](https://eslint.org/docs/rules/arrow-parens) rule.

**Review by makkoncept — CHANGES_REQUESTED** [review]
hello @manisacharya, 
I think you have not checked the git diff correctly before committing, as you have also pushed style changes on the whole file(maybe due to autoformatting on your local machine). You can read about project contribution guidelines in [INSTRUCTIONS.md](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/INSTRUCTIONS.md).

Please rectify the issues. 
Thanks!

File: src/options/options.js (hunk 2 above)
Comment (manisacharya) [reply src/options/options.js]: I will update this format then @makkoncept

**Comment by manisacharya** [comment]
> hello @manisacharya,
> I think you have not checked the git diff correctly before committing, as you have also pushed style changes on the whole file(maybe due to autoformatting on your local machine). You can read about project contribution guidelines in [INSTRUCTIONS.md](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/INSTRUCTIONS.md).
> 
> Please rectify the issues.
> Thanks!

Thanks, @makkoncept for the feedback. I have reverted the format again. Can you verify this again?

**Review by makkoncept — APPROVED** [review]
Looks good!

**Pull request merged**